build/NuGetPackages/Microsoft.Build.Runtime.nuspec

```diff
@@ -0,0 +1,87 @@
+    <file src="$configuration$\Output\MSBuild.exe.config" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Common.CrossTargeting.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Common.CurrentVersion.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Common.overridetasks" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Common.props" target="contentFiles\any\net46\$targetMSBuildToolsVersion$\" />
+    <file src="$configuration$\Output\Microsoft.Common.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Common.tasks" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.CSharp.CurrentVersion.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.CSharp.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Data.Entity.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.NETFramework.CurrentVersion.props" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.NETFramework.CurrentVersion.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.NETFramework.props" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.NETFramework.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.ServiceModel.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.VisualBasic.CurrentVersion.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.VisualBasic.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.VisualStudioVersion.v11.Common.props" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.VisualStudioVersion.v12.Common.props" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.VisualStudioVersion.v14.Common.props" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.WinFx.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.WorkflowBuildExtensions.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Microsoft.Xaml.targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Workflow.Targets" target="contentFiles\any\net46\" />
+    <file src="$configuration$\Output\Workflow.VisualBasic.Targets" target="contentFiles\any\net46\" />
+
+    <!--
+      contentFiles\any\netcoreapp1.0
+    -->
+    <file src="$configuration$-NetCore\Output\MSBuild.exe" target="contentFiles\any\netcoreapp1.0\" />
```

Comment:
I'll address this in a future commit.  For now, this will unblock apps which want to do project evaluation and in-process builds but they won't be able to shell out to `MSBuild.exe`